src/test/statistics/statistics_import_export_test.cpp

```diff
@@ -11,7 +12,9 @@
 
 namespace opossum {
 
-class StatisticsImportExportTest : public ::testing::Test {};
+class StatisticsImportExportTest : public ::testing::Test {
+  void TearDown() { std::remove((test_data_path + "exported_table_statistics_test.json").c_str()); }
```

Comment:
y is this necessary? I thought test_data_path makes sure there are no conflicts with filenames and such?

Reply:
1) `test_data_path` is not necessarily set. If you call `./build/hyriseTest` on the current master, you will end up with the json file in your project root.
2) It isolates only different builds (mainly for CI). It does not clear that folder.
3) In any case, each test should clean up after itself.

Reply:
woah, so many reasons